auth_saml/tests/test_pysaml.py

```diff
@@ -39,6 +47,44 @@ def setUp(self):
 
         self.idp = FakeIDP([self.saml_provider._metadata_string()])
 
+        # Create a user with only password, and another with both password and saml id
+        self.user, self.user2 = (
+            self.env["res.users"]
+            .with_context(no_reset_password=True)
```

Comment:
```suggestion
            .with_context(no_reset_password=True, tracking_disable=True)
```

Reply:
Also changed in test_disallow_user_password_no_password_set.